interfaces/builtin/audio_playback.go

```diff
@@ -109,6 +110,21 @@ owner /{,var/}run/pulse/** rwk,
 owner /run/pulse/native/ rwk,
 owner /run/user/[0-9]*/ r,
 owner /run/user/[0-9]*/pulse/ rw,
+
+# DBus accesses
+include <abstractions/dbus-strict>
+
+# Allow binding the service to the requested connection name
+dbus (bind)
+    bus=system
+    name="org.pulseaudio.Server",
+dbus (send)
+   bus=system
+   path=/org/freedesktop/DBus
+   interface=org.freedesktop.DBus
+   member={Request,Release}Name
+   peer=(name=org.freedesktop.DBus, label=unconfined),
+
```

Comment:
We probably should have some rules for unconfined clients to connect to this service, no?

Reply:
Right, I think that using the dbus interface will solve all these comments, removing the dbus stuff now.